.travis.yml

```diff
@@ -62,9 +62,10 @@ jobs:
       install: skip
       script:
         - ./.travis/setup_database &
-        - ./gradlew --parallel test integTest
+        - ./gradlew --parallel test integTest jacocoTestReport
+        - bash <(curl -s https://codecov.io/bash)  # upload coverage report - https://github.com/codecov/example-gradle
     - stage: verify
-      env: "JDK12 Unit Tests"
+      env: "Unit Tests"
```

Comment:
The unit and integ tests take 11 minutes, the rest of the stages in verify take about 7. 
If we remove the JDK12 tests, I think we would probably benefit to stay with 4 concurrent stages and move the integ tests to their own stage to help reduce that 11 minute runtime.